docs/index.bs

```diff
@@ -2926,7 +2934,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           The |task| *must* use |activeWorker|'s [=event loop=] and the [=handle functional event task source=].
 
       1. Wait for |task| to have executed or been discarded.
-      1. If the time difference in seconds calculated by the current time minus |registration|'s [=last update check time=] is greater than 86400, invoke [=Soft Update=] algorithm with |registration|.
+      1. If |registration| is [=stale=], then run the [=Soft Update=] algorithm with |registration|.
```

Comment:
Should this also be in parallel?

Reply:
Done. I noticed this too and thought I'd just keep the existing behavior, but yea it looks right to do it in parallel. I'm also wondering if there's a point to "Wait for |task| to have executed..." but will leave it for now.